lib/cloudregister/registerutils.py

```diff
@@ -1635,3 +1644,25 @@ def __replace_url_target(config_files, new_smt):
                     current_service_server,
                     new_smt.get_FQDN())
                 )
+
+
+# -----------------------------------------------------------------------------
+def __connection_check(smt_ip_addr, ip_format):
```

Comment:
`ip_format` is not required as an argument. This is a simple check that can be performed in the function.